core/templates/dev/head/domain/statistics/LearnerActionObjectFactorySpec.js

```diff
@@ -21,14 +21,23 @@ describe('Learner Action Object Factory', function() {
 
   beforeEach(inject(function($injector) {
     this.laof = $injector.get('LearnerActionObjectFactory');
+    this.LEARNER_ACTION_SCHEMA_LATEST_VERSION =
+      $injector.get('LEARNER_ACTION_SCHEMA_LATEST_VERSION');
   }));
 
   it('should create a new learner action', function() {
-    var learnerActionObject = this.laof.createNew('AnswerSubmit', {}, 1);
+    var learnerActionObject = this.laof.createNew('AnswerSubmit', {});
 
     expect(learnerActionObject.actionType).toEqual('AnswerSubmit');
     expect(learnerActionObject.actionCustomizationArgs).toEqual({});
-    expect(learnerActionObject.schemaVersion).toEqual(1);
+    expect(learnerActionObject.schemaVersion)
+      .toEqual(this.LEARNER_ACTION_SCHEMA_LATEST_VERSION);
+  });
+
+  it('should use a specific schema version if provided', function() {
+    var learnerActionObject = this.laof.createNew('AnswerSubmit', {}, -5);
```

Comment:
Do we want to validate that the schema version is positive and less than LEARNER_ACTION_SCHEMA_LATEST_VERSION? 

Other numbers should ideally be invalid right?